src/api/route-services/gem2s.js

```diff
@@ -66,6 +66,7 @@ class Gem2sService {
       input: { type: experiment.meta.type },
       sampleIds: experiment.sampleIds,
       sampleNames: experiment.sampleIds.map((sampleId) => samples[sampleId].name),
+      authJWT,
```

Comment:
Do we actually use this authJWT in gem2s?

Reply:
No, we just pass it so we get it back in the last gem2s step so we caa then pass it to qc.

I consulted with marcell and we ended up doing this implementation, the alternative I could think of was storing it somewhere until gem2s finished which sounds worse than this.